website/src/pages/index.js

```diff
@@ -0,0 +1,229 @@
+      <div className="container">
+        <div className="row">
+          <div className={clsx('col text--center')}>
+            <h2>Contributed by the community in collaboration with</h2>
+          </div>
+        </div>
+        <div className="row">
+          <div className={clsx('col col--3 text--center')}>
+            <a href="https://cloud.google.com/">
+              <img className={styles.companySvg} src="img/google_cloud_logo_vector.svg" />
+            </a>
+          </div>
+          <div className={clsx('col col--3 text--center')}>
+            <a href="https://www.microsoft.com/">
+              <img className={styles.companySvg} src="img/microsoft_logo.svg" />
+            </a>
+          </div>
+          <div className={clsx('col col--3 text--center')}>
+            <a href="https://www.styra.com/">
+              <img className={styles.companySvg} src="img/styra_logo-blue.svg" />
+            </a>
+          </div>
+          <div className={clsx('col col--3 text--center')}>
+            <a href="https://www.cncf.io/">
+              <img className={styles.companySvg} src="img/cncf_logo.svg" />
+            </a>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+const FeatureList = [
+  {
+    title: 'Kubernetes Native',
+    Svg: require('@site/static/img/kubernetes_logo.svg').default,
+    description: (
+      <>
+        Gatekeeper makes managing policies on top of Kubernetes easy.
+        Policies can be enforced at admission time or at runtime via
+        the audit functionality.
+      </>
+    ),
+  },
+  {
+    title: 'Powered by Open Policy Agent',
+    Svg: require('@site/static/img/opa_logo.svg').default,
+    description: (
+      <>
+        Gatekeeper is powered by the Open Policy Agent (OPA) project.
+        Using OPA allows you to write policies that are powerful, flexible,
+        and portable.
+      </>
+    ),
+  },
+  {
+    title: 'Extensive Policy Library',
+    Svg: require('@site/static/img/library.svg').default,
+    description: (
+      <>
+        Browse the policy library to find existing policies that fit
+        your use case. Each policy in the library can be extended and 
+        customized to fit your needs.
+      </>
+    ),
+  },
+];
+
+function Feature({Svg, title, description}) {
+  return (
+    <div className={clsx('col col--4')}>
+      <div className="text--center">
+        <Svg className={styles.featureSvg} role="img" />
+      </div>
+      <div className="text--center padding-horiz--md">
+        <h3>{title}</h3>
+        <p>{description}</p>
+      </div>
+    </div>
+  );
+}
+
+function HomepageFeatures() {
+  return (
+    <section className={styles.features}>
+      <div className="container">
+        <div className="row">
+          {FeatureList.map((props, idx) => (
+            <Feature key={idx} {...props} />
+          ))}
+        </div>
+      </div>
+    </section>
+  );
+}
+
+const Services = [
```

Comment:
nit: Can we make these boxes similar in size?

![image](https://user-images.githubusercontent.com/1626598/234710695-62faaf34-88e3-4d72-93ec-057acd9bab00.png)

Reply:
Sure. Could you open an issue to track it?